.projenrc.ts

```diff
@@ -78,8 +78,4 @@ project.npmignore?.exclude('/scripts/', '.projenrc.ts');
 // cdklabs-projen-project-types is overzealous about adding this dependency
 project.deps.removeDependency('constructs');
 
-// temporary to
-// to resolve https://github.com/aws/jsii/issues/4658
```

Comment:
this has been resolved in so much as we are pinning typeguard to `<4.3.0` in `jsii-pacmak`